cypress/integration/dashboard_spec.ts

```diff
@@ -42,11 +43,23 @@ describe('The Dashboard', () => {
     cy.getBySel('swap-assets-button').click()
     // cy.getBySel('trade-rate-quote').should('have.text', '1 FOX = 0.00011 ETH') // TODO - mock response and test here
     cy.getBySel('trade-preview-button').should('be.disabled')
+
+    // Clicking 'swap-assets-button' will cause XHR requests, lets wait for
+    // those requests to complete since the 'trade-preview-button' will have
+    // the text "Loading..." (displays a spinner)
+    // cy.intercept({ method: 'GET', url: '/swap/**'}).as('getSwap')
+    // cy.intercept({ method: 'GET', url: '**/account/**'}).as('getAccount')
+    // cy.wait(['@getSwap', '@getAccount'], { timeout: 60000 })
```

Comment:
Are these comments intentionally here to describe what `waitForAllGetReqs` is doing behind the scenes, or are they a stale artefact?

Reply:
stale 🙃